### PR TITLE
Update to Outdated files in the dev-1.18-ko.2 branch.

### PR DESCRIPTION
**Pull request by ysyukr**
### 31 of 32 files to be modified

- [x] content/en/_index.html
- [x] content/en/docs/concepts/architecture/controller.md
- [x] content/en/docs/concepts/architecture/master-node-communication.md
- [x] content/en/docs/concepts/architecture/nodes.md
- [x] content/en/docs/concepts/cluster-administration/cluster-administration-overview.md
- [x] content/en/docs/concepts/overview/what-is-kubernetes.md
- [x] content/en/docs/concepts/overview/working-with-objects/namespaces.md
- [x] content/en/docs/concepts/policy/limit-range.md
> full url을 짧은 링크로 변경하는 #20221 PR 제출 및 해당 내용 반영했습니다.
- [x] content/en/docs/concepts/scheduling/scheduler-perf-tuning.md
- [x] content/en/docs/concepts/services-networking/endpoint-slices.md
- [x] content/en/docs/concepts/services-networking/ingress.md
- [x] content/en/docs/concepts/services-networking/service.md
- [x] content/en/docs/concepts/storage/persistent-volumes.md
- [x] content/en/docs/concepts/workloads/controllers/deployment.md
- [x] content/en/docs/concepts/workloads/controllers/replicaset.md
- [x] content/en/docs/concepts/workloads/controllers/replicationcontroller.md
- [x] content/en/docs/concepts/workloads/pods/init-containers.md
- [x] content/en/docs/contribute/_index.md
- [x] content/en/docs/contribute/participating.md
- [x] content/en/docs/contribute/style/write-new-topic.md
- [x] content/en/docs/reference/glossary/container-runtime.md
- [x] content/en/docs/reference/issues-security/security.md
- [x] content/en/docs/reference/kubectl/cheatsheet.md
- [x] content/en/docs/reference/tools.md
- [x] content/en/docs/setup/_index.md
- [x] content/en/docs/setup/learning-environment/minikube.md
- [x] content/en/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md
- [ ] content/en/docs/tasks/debug-application-cluster/resource-metrics-pipeline.md
> 번역시 의미 변화가 없어 미반영
- [x] content/en/docs/tasks/run-application/horizontal-pod-autoscale.md
- [x] content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html
- [x] content/en/docs/tutorials/stateful-application/cassandra.md
- [x] content/en/training/_index.html

from #20209 
/language ko

### CODE REVIEW

File: content/ko/docs/tutorials/stateful-application/cassandra.md [hunk 1]
```diff
@@ -105,12 +102,21 @@ cassandra   ClusterIP   None         <none>        9042/TCP   45s
...
-1. `cassandra-statefulset.yaml` 파일로 카산드라 스테이트풀셋 생성
+`cassandra-statefulset.yaml` 파일로 카산드라 스테이트풀셋 생성
+
+```shell
+# cassandra-statefulset.yaml를 수정하지 않은 경우에 이것을 사용한다.
+kubectl apply -f https://k8s.io/examples/application/cassandra/cassandra-statefulset.yaml
+```
+
+클러스터에 맞게 `cassandra-statefulset.yaml` 를 수정해야하는 경우 다음을 다운로드 한 다음
+수정된 버전을 저장 한 폴더에서 해당 매니페스트를 적용한다.
```
Comment (seokho-son) [inline content/ko/docs/tutorials/stateful-application/cassandra.md:113]:
```suggestion
수정된 버전을 저장한 폴더에서 해당 매니페스트를 적용한다.
```

File: content/ko/docs/tutorials/stateful-application/cassandra.md [hunk 2]
```diff
@@ -105,12 +102,21 @@ cassandra   ClusterIP   None         <none>        9042/TCP   45s
...
+https://k8s.io/examples/application/cassandra/cassandra-statefulset.yaml
+```shell
+# cassandra-statefulset.yaml를 로컬에서 수정한 경우에 사용한다.
```
Comment (seokho-son) [inline content/ko/docs/tutorials/stateful-application/cassandra.md:116]:
```suggestion
# cassandra-statefulset.yaml을 로컬에서 수정한 경우에 사용한다.
```

File: content/ko/docs/tutorials/stateful-application/cassandra.md [hunk 3]
```diff
@@ -120,7 +126,7 @@ cassandra   ClusterIP   None         <none>        9042/TCP   45s
     kubectl get statefulset cassandra
     ```
 
-    응답은 다음과 같다.
+    응답은 다음 유사하다.
```
Comment (seokho-son) [inline content/ko/docs/tutorials/stateful-application/cassandra.md:129]:
```suggestion
    응답은 다음과 유사하다.
```

File: content/ko/docs/tutorials/stateful-application/cassandra.md [hunk 4]
```diff
@@ -135,15 +141,16 @@ cassandra   ClusterIP   None         <none>        9042/TCP   45s
     kubectl get pods -l="app=cassandra"
     ```
 
-    응답은 다음과 같다.
+    응답은 다음 유사하다.
```
Comment (seokho-son) [inline content/ko/docs/tutorials/stateful-application/cassandra.md:144]:
```suggestion
    응답은 다음과 유사하다.
```

File: content/ko/docs/tutorials/stateful-application/cassandra.md [hunk 5]
```diff
@@ -135,15 +141,16 @@ cassandra   ClusterIP   None         <none>        9042/TCP   45s
...
 
     ```shell
     NAME          READY     STATUS              RESTARTS   AGE
     cassandra-0   1/1       Running             0          1m
     cassandra-1   0/1       ContainerCreating   0          8s
     ```
 
-    모든 3개 파드가 배포되기까지 몇 분이 소요될 수 있다. 배포 후에는 같은 명령은 다음같이 응답한다.
+    모든 3개 파드가 배포되기까지 몇 분이 소요될 수 있다. 배포 후에는 같은 명령은 다음같이 유사하게
```
Comment (seokho-son) [inline content/ko/docs/tutorials/stateful-application/cassandra.md:152]:
```suggestion
    모든 3개 파드가 배포되기까지 몇 분이 소요될 수 있다. 배포 후, 동일 명령은 다음과 유사하게
```

File: content/ko/docs/tutorials/stateful-application/cassandra.md [hunk 6]
```diff
@@ -181,14 +189,14 @@ cassandra   ClusterIP   None         <none>        9042/TCP   45s
     kubectl edit statefulset cassandra
     ```
 
-    이 명령은 터미널에서 편집기를 연다. 변경해야할 행은 `replicas` 필드이다. 다음 예제는 `StatefulSet` 파일에서 발췌했다.
+    이 명령은 터미널에서 편집기를 연다. 변경해야할 행은 `replicas` 필드이다. 다음 예제는 StatefulSet 파일에서 발췌했다.
```
Comment (seokho-son) [inline content/ko/docs/tutorials/stateful-application/cassandra.md:192]:
```suggestion
    이 명령은 터미널에서 편집기를 연다. 변경해야할 행은 `replicas` 필드이다. 다음 예제는 스테이트풀셋 파일에서 발췌했다.
```

File: content/ko/docs/tutorials/stateful-application/cassandra.md [hunk 7]
```diff
@@ -205,15 +213,15 @@ cassandra   ClusterIP   None         <none>        9042/TCP   45s
 
 1. 레플리카 개수를 4로 바꾸고, 매니페스트를 저장한다.
 
-    The `StatefulSet` now contains 4 Pods.
+    The StatefulSet now scales to run with 4 Pods.
```
Comment (seokho-son) [inline content/ko/docs/tutorials/stateful-application/cassandra.md:216]:
```suggestion
    스테이트풀셋은 4개의 파드를 실행하기 위해 스케일 한다.
```

**Comment by ysyukr** [comment]
@seokho-son 리뷰 감사합니다.
확인 및 반영했습니다. :)

File: content/ko/docs/contribute/participating.md [hunk 8]
```diff
@@ -307,8 +308,9 @@ PR 소유자에게 조언하는데 활용된다.
 
 쿠버네티스 문서화에 기여하는 일에 대한 보다 많은 정보는 다음 문서를 참고한다.
 
-- [기여 시작하기](/docs/contribute/start/)
-- [문서 스타일](/docs/contribute/style/)
+- [신규 컨텐츠 기여하기](/docs/contribute/overview/)
```
Comment (seokho-son) [inline content/ko/docs/contribute/participating.md:311]: @ysyukr 
현재 PR을 기준으로, 해당 링크에 해당하는 문서가 없는 것을 발견하였습니다.
현재 master에는 해당 문서가 있습니다. 향후 작업 브랜치가 master에 머지되면 문제는 없겠지만, 
혹시 outdated에 해당하는 문서가 아닌지 확인 한번 부탁 드립니다.

Comment (ysyukr) [reply content/ko/docs/contribute/participating.md:311]: `git diff upstream/dev-1.18-ko.1 upstream/dev-1.18-ko.2 -- content/en/docs/contribute/participating.md` 시 정상적인 outdated 파일은 맞는 것으로 확인됩니다.
다만, 원문 확인시에도 링크에 문제가 있는 것으로 보아 정상적인 링크는 `/docs/contribute/new-content/overview/` 로 확인됩니다.

**Pull request merged**